ClientApp/src/Pages/GettingStarted/GettingStarted.scss

```diff
@@ -0,0 +1,60 @@
+
+    .image-and-text {
+        display: flex;
+        align-self: center;
+        flex-direction: row;
+        width: 100%;
+        justify-content: center;
+        align-items: center;
+    }
+
+    .getting-started {
+        text-align: center;
+    }
+
+    .selected-icon {
+        padding-left: 5%;
```

Comment:
I think this would better as something like 
```
.icon { 
     padding-left: 5%;
      padding-right: 5%;
      font-size: x-large;
}
.selected-icon {
    background-color: orange;
}
```